### ISSUE TITLE
MacBook Pro 13"/15" 2018

### ISSUE THREAD

**Issue by faken** [ashfurrow/xcode-hardware-performance#70]
I'm really wondering how much faster are the newly refreshed MacBook Pros really are. 
Would be nice to have them listed here!

**Comment by beeradmoore** [comment]
Curious how that i9 compares to an i7 due to [this](https://www.youtube.com/watch?v=Dx8J125s4cg). I assume compiling code would not thermal throttle it enough to see compared to their 30min encode times.

**Comment by ZevEisenberg** [comment]
The throttling issue is (supposedly) [fixed](https://www.macrumors.com/2018/07/24/2018-macbook-pro-throttling-bug-fix/) via a software update.

**Comment by beeradmoore** [comment]
Still curious if after the patch, the same test that @joesanford committed would have any different results, or if there I no noticeable difference or throttling for compiling code.

**Comment by Buju77** [comment]
yeah, i'm also interested in the compiling different between the new i7 and i9 15" MBP. If it's worth the money upgrade to go with i9

**Comment by joesanford** [comment]
I'll re-run the tests soon and submit a PR if the results are any different.

**Comment by joesanford** [comment]
Sorry for the delay, but PR opened for new i9 2018 timings https://github.com/ashfurrow/xcode-hardware-performance/pull/76

**Comment by benstahl** [comment]
The difference between the 2.9 i9 and the 2.2 i7 15" seem way more extreme than almost every benchmark I've seen for any software/workflow. For almost anything else the difference is usually less than 10%, sometimes much less. But these tests are showing the i9 40% faster (fresh build) and 225% faster (incremental build).

**Comment by iby** [comment]
There are two tests for MacBook Pro 2.9 GHz i9, both looking crazy and beating the 10-core iMac Pro. Assuming it's in line with everyone else's results, is there any explanation why and how? While compiling too-many-source-files project on an older 2.8 GHz i7 I always get a feeling like I need 12 more cores. But these results pretty much say just get the latest MBP and don't waste money on the iMac Pro. I'm confused… 😕

**Comment by fastred** [comment]
@ianbytchek IMHO iMac Pro and MBP 2018" results can't be compared directly as-is. They use different:
- commit version
- Xcode version
- build system
